src/renderer/actions/register.ts

```diff
@@ -65,32 +65,40 @@ export interface IWithdrawalKeyAction extends Action<RegisterActionTypes> {
 
 // After password action
 export const afterPasswordAction = (password: string) => {
-    return (dispatch: Dispatch<Action<RegisterActionTypes>>, getState: () => IRootState): void => {
+    return async (dispatch: Dispatch<Action<RegisterActionTypes>>, getState: () => IRootState): Promise<void> => {
         // 1. Save to keystore
-        const signingKey = getState().register.signingKey;
-        V4Keystore.create(
-            `${KEYSTORE_DEFAULT_DIRECTORY}/${getV4Filename()}.json`, 
-            password, new Keypair(PrivateKey.fromHexString(signingKey))
+        dispatch(startRegistrationSubmission());
+        const signingKey = PrivateKey.fromBytes(
+            Buffer.from(getState().register.signingKey.slice(2), "hex")
+        );
+        const accountDirectory = path.join(getConfig().storage.accountsDir, "default");
```

Comment:
maybe we should export account handling (location and keystore management) somewhere else? I guess we will probably use it for login and unlocking.

Reply:
also not using DEFAULT_ACCOUNT var

Reply:
> maybe we should export account handling (location and keystore management) somewhere else? I guess we will probably use it for login and unlocking.

Hmm not sure where else you would use account and keystore creation though. I guess we can see later on